api/src/main/java/org/geysermc/geyser/api/GeyserApi.java

```diff
@@ -134,6 +136,30 @@ public interface GeyserApi extends GeyserApiBase {
     @NonNull
     PlatformType platformType();
 
+    /**
+     * Gets the version of Java Minecraft that is supported.
+     *
+     * @return the supported version of Java Minecraft
+     */
```

Comment:
If we were to follow the Google Style Guide it'd be fine to simplify this to:
`Returns the version of Java Minecraft that is supported`

Reply:
Not against doing that, i've mostly mirrored the style of existing method docs